compiler/rustc_span/src/hygiene.rs

```diff
@@ -903,6 +903,21 @@ impl Span {
     }
 }
 
+bitflags::bitflags! {
+    /// Flags for the syntax extension/expansion (e.g. should debuginfo be collapsed? should the
+    /// `unsafe_code` lint be suppressed?).
+    #[derive(Encodable, Decodable, HashStable_Generic)]
+    pub struct ExpnFlags: u32 {
```

Comment:
One byte should be enough?
```suggestion
    pub struct ExpnFlags: u8 {
```

Reply:
Yeah, I think so, fixed.